example/spec/parallel/ingest/ingestPdrWithNodeNameSpec.js

```diff
@@ -512,7 +508,7 @@ describe('Ingesting from PDR', () => {
 
             // the output of the CheckStatus is used to determine the task of choice
             const checkStatusTaskName = 'CheckStatus';
-            const successStepName = 'WorkflowSucceeded';
+            const successStepName = 'SendPAN';
```

Comment:
Similar to ingestFromPdrSpec, should we have some assertions that the SendPAN task was invoked and successful?